docs/changes/ATHLETE_SEASON_UPDATE_LOGIC.md

```
@@ -0,0 +1,238 @@
+    }
+}
+```
+
+**Purpose**: Publishes `ProcessImageRequest` event for athlete season headshots, allowing image sourcing pipeline to download and store the image.
+
+---
+
+## ?? **Processing Flow**
+
+### **New AthleteSeason**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)